package-lock.json

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
we don't need/want a `package-lock` (this is from npm) because we are using yarn